iModelCore/ecobjects/PublicApi/ECObjects/ECSchema.h

```diff
@@ -2466,7 +2466,7 @@ struct SchemaKeyLessThan
 };
 
 typedef bmap<SchemaKey , ECSchemaPtr> SchemaMap;
-typedef std::function<bool(const std::pair<SchemaKey, ECSchemaPtr>)> SchemaKeyMatchFnPredicate;
+typedef std::function<bool(SchemaKeyCR)> SchemaKeyMatchFn;

```

Comment:
Just very minor. I checked our codebase and we never use the Fn suffix. Let's not introduce it now. Some existing functions use xCallback, xPredicate (though there already is a SchemaKeyMatchPredicate) or xHandler but Handler is not a good match here. I suggest "SchemaKeyMatchCallback".